docs/index.md

```diff
@@ -0,0 +1,397 @@
+---
+description: pyfreethreading is a centralized collection of documentation and trackers around compatibility with
+the free-threaded build of CPython in the PyData ecosystem
+---
+
+## Improving Ecosystem Compatibility with Free-Threaded Python
+
+Quansight Labs is working with the Python runtime team at Meta and stakeholders
+across the ecosystem to jumpstart work on converting the libraries that make up
+the scientific Python and AI/ML stacks to work with the free-threaded (nogil)
+build of CPython 3.13. Additionally, we will look at libraries like PyO3
+that are needed to interface with CPython from other languages.
+
+Our initial goal is to ensure libraries at the bottom of the stack like
+NumPy, pybind11, and Cython are usable with free-threaded CPython. We will also
+be updating packaging tools like meson-python needed to support building wheels
+for free-threaded CPython. Once those tools and libraries are in a stable
+enough state, we will begin looking at libraries higher in the stack.
+
+### What is this repository?
+
+This repository is for coordinating ecosystem-wide work. We will use
+this repository to track, understand, and provide documentation for
+dealing with issues that we find are common across many
+libraries. Issues that are specific to a project should be reported in
+that project's issue tracker.
```

Comment:
Let's keep everything above this line in the readme and let's keep everything below only in this file. We can add a new intro section for the website and link to the website from the readme.

Reply:
(I know you said we should iterate later, I'd just prefer not to have duplicated content right after we merge this)

Reply:
Agreed! Done.